examples/demos/rendering.js

```diff
@@ -39,21 +42,56 @@ const customSlotPropGetter = date => {
   else return {}
 }
 
-let Rendering = ({ localizer }) => (
-  <Calendar
-    events={events}
-    localizer={localizer}
-    defaultDate={new Date(2015, 3, 1)}
-    defaultView={Views.AGENDA}
-    dayPropGetter={customDayPropGetter}
-    slotPropGetter={customSlotPropGetter}
-    components={{
-      event: Event,
-      agenda: {
-        event: EventAgenda,
-      },
-    }}
-  />
-)
+const ColoredEventWrapper = ({ children, event }) => {
+  console.log()
+  return React.cloneElement(Children.only(children), {
```

Comment:
I think we should have a check for children.only before clone, and throw warning when in development, the message will help other devs know that this component cant receive multiple children

Reply:
This project isn't written in typescript so it's actually difficult to do that. But i will try to find a way to do it

Reply:
I mean we have an notify message like this.
```ts
   if (process.env.NODE_ENV !== "production") {
      if (React.Children.toArray(children).length > 1) { //or you can try catch the Children.only
         throw new Error("This <ComponentName> can only receive 1 children")
      }
   }
```
then you can remove the React.only in the render part

Reply:
It is a common pattern that is frequently used in some component library